src/components/PersonItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={person.slug === slug ? 'has-background-warning' : ''}
+    >
+      <td>
+        <Link
+          to={{
+            pathname: `${person.slug}`,
+            search: searchParams.toString(),
+          }}
+          className={person.sex === 'f' ? 'has-text-danger' : ''}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {person.mother ? (
+          <Link to={person.mother.slug} className="has-text-danger">
+            {person.motherName}
+          </Link>
+        ) : (
+          person.motherName || '-'
+        )}
+      </td>
+      <td>
+        {person.father ? (
+          <Link to={person.father.slug}>{person.father.name}</Link>
```

Comment:
The `Link` to the father's profile should use `person.father.slug` instead of `person.father.name` for correct URL formation. This ensures that the link points to the correct route.